server/schedulers/hot_region_config.go

```diff
@@ -82,6 +83,7 @@ type hotRegionSchedulerConfig struct {
 	DstToleranceRatio      float64  `json:"dst-tolerance-ratio"`
 	ReadPriorities         []string `json:"read-priorities"`
 	WritePriorities        []string `json:"write-priorities"`
+	StrictPickingStore     bool     `json:"strict-picking-store"`
```

Comment:
I think it should be `json:"strict-picking-store,string"`, Do you test it with `pd-ctl`? BTW, It required 2LGT.